### DISCUSSION TITLE
Reports not generated in the user's language (Portuguese)

### DISCUSSION THREAD

**Discussion by Dimas-013** [monarc-project/MonarcAppFO#536] [General]
Hello, 
The Portuguese language option doesn't seem to appear when creating a model even if the front end is in Portuguese. 
It would be great if the database could be updated so that the report can be generated in Portuguese.

 Using Monarc v2.12.7.

**Comment by ruslanbaidan** [comment]
There are 2 notions of languages in the system. The user interface and analysis languages.
There are only 4 available languages for the analysis (FR, EN, DE, NL), but one of them can be replaces manually to the required one.
For your needs, you could download reports in e.g. English (if your analysis is in EN), translate it and upload as an English one.
The only thing is that all the report data will be as they are in the database of the analysis (e.g. assets, threats, vulnerabilities etc are in English if the analysis is in English).

**Reply by danivraposo** [reply]
How do i change manually the language? which file do i need to change?

**Reply by ruslanbaidan** [reply]
There is no particular file to be changed. The data are stored in the database, under e.g. label1 (FR), label2 (EN), label3 (DE), label4 (NL), name1, name2, etc. It's possible to changed the values in the application or directly in the database.

For the initial installation (if a local instance on Monarc is installed on a machine), it's also possible to update all the values for a certain language in the SQL [here](https://raw.githubusercontent.com/monarc-project/MonarcAppFO/master/db-bootstrap/monarc_data.sql). Meaning for example all the values of label1, name1, description1, path1 (FR) etc are translated/replaced to the desired language here and imported in the empty db (within the default installation process).

**Reply by danivraposo** [reply]
And in this file i have to do the transalation or the same are there already?
